src/sync.cpp

```diff
@@ -3393,6 +3393,11 @@ bool JSONSyncConfigIOContext::deserialize(SyncConfig& config, JSON& reader) cons
 
         case TYPE_TARGET_HANDLE:
             config.mRemoteNode = reader.gethandle(MegaClient::NODEHANDLE);
+            if ((config.mRemoteNode & 0xFFFFFFFFFFFF) == (UNDEF & 0xFFFFFFFFFFFF))
+            {
+                // we can have a much nicer solution when NodeHandle is merged from the sync rework branch
+                config.mRemoteNode = UNDEF;
+            }
```

Comment:
Actually, `mRemoteNode` is not initialized when it reaches the `gethandle()`. If we initialize it to `UNDEF` in the declaration of the variable, when we reach this point, we could test for `if (!ISUNDEF(config.mRemoteNode))` and it will be enough, won't it?
If you have created a new class NodeHandle to avoid the issues of handles of 6 bytes, I hope that all refactoring is targeting `develop` as an enhancement / maintenance :crossed_fingers: